kernel/kernel-api/src/main/java/io/delta/kernel/internal/tablefeatures/TableFeatures.java

```diff
@@ -39,6 +40,18 @@
 /** Contains utility methods related to the Delta table feature support in protocol. */
 public class TableFeatures {
 
+  /**
+   * The prefix for setting an override of a feature option in {@linkplain Metadata} configuration.
```

Comment:
I think there could be a better name than "setting an override"?

`SET_TABLE_FEATURE_SUPPORTED_PREFIX`?